### ISSUE TITLE
Do I need to adjust lr when batch size changes?

### ISSUE THREAD

**Issue by JeffWang987** [open-mmlab/mmdetection3d#574]
The SECOND config for Kitti uses batch size 6. However, in my config, I change batch size to 4. 
Do I need to simultaneously change the base learning rate? If so, any general advise?

Thx in advance!

**Comment by THU17cyz** [comment]
Hi @JeffWang987 , 

Please refer to the linear scaling [here](https://mmdetection3d.readthedocs.io/en/latest/1_exist_data_model.html#train-predefined-models-on-standard-datasets).
